projects/templates/projects/done_test.html

```diff
@@ -16,7 +16,39 @@
 {% block content %}
     <div >
         <p>Project</p>
+        <li>{{ project_data.id }}</li>
+        <li>{{ project_data.name }}</li>
+        <li>{{ project_data.description }}</li>
+        <li>{{ project_data.due_date }}</li>
+        {% if project_data.status == 0 %}
```

Comment:
Instead of checking the integer value manually, you can just display the status value directly. In the future, if another status was added you wouldn't want to go throughout all your code and add another "if" statement.

Documentation:
https://docs.djangoproject.com/en/3.0/ref/models/instances/#django.db.models.Model.get_FOO_display

Example:
https://github.com/craigkelleher/CS361-Team-10-Project-repository/blob/master/projects/templates/projects/projects.html